samples/net/full_mqtt_tls/README.rst

```diff
@@ -0,0 +1,44 @@
+.. _full-mqtt-tls-sample:
+
+Full MQTT TLS Stack Sample
+#################
+
+Overview
+********
+
+This sample application demonstrates a "full stack" application.  This
+currently is able to
+
+- Aquire a DHCPv4 lease.
+- Connect to an SNTP server and acquire current time
+- Establish a TLS connection with the Google IOT Cloud servers
+- Publish data to the Google IOT Cloud
+- Send/Receive keep alive / pings from cloud server
+
+The source code for this sample application can be found at:
+:file:`samples/net/full_mqtt_tls`.
+
+Requirements
+************
+- Entropy source
+- Google IOT Cloud account
+- Google IOT Cloud credentials and required information
+- Network connectivity
+
+Building and Running
+********************
+This application has been built and tested on the NXP FRDMK64F.  RSA or
+ECDSA certs/keys are required to authenticate to the Google IOT Cloud.
+The application includes a key creation script.  The create_keys.py script
+will need to be run in the `samples/net/full_mqtt_tls/src/private_info/`
+directory.
+
+Users will also be required to configure the following Kconfig options
+based on their Google Cloud IOT project:
+- CLOUD_CLIENT_ID - created from <telemetry>/<region>/registries/<registry name>/devices/<device name>
```

Comment:
Add a blank line before the first item in a list (otherwise it's not recognized as a list, and will be included with the previous sentence.)

Reply:
done